pyof/foundation/basic_types.py

```diff
@@ -237,18 +242,20 @@ class IPAddress(GenericType):
     netmask = UBInt32()
     max_prefix = UBInt32(32)
 
-    def __init__(self, address="0.0.0.0/32"):
+    def __init__(self, value=None):
```

Comment:
I agree this is only strictly needed in cases where the init object is mutable,
but this is not about "looking the same", it is about consistency.

Almost all classes we implemented receive named parameters initially set to ``None`` in the call, making it explicit like this ensures and remembers that calling ``IPAddress()`` and ``IPAdress(None)`` will forcibly behave the same.
That said, the important part here is actually line 251 (https://github.com/kytos/python-openflow/pull/427/files#diff-83e119f254a95803fa6cdaff8bd5bee3R251), which ensures the class can be called with ``None``, something that is not possible right now.

We could to make it accept ``None`` on line 251, and still leave the default value being set on 245, but that will only make it necessary to define the default value twice, since it'd still have to be set in line 252 for the cases where the call is made with None. So I think that would be redundant and just unnecessary.